sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,79 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+
+    defNum = 1
+
+    defLast = False
+    
+    hasPrinted = False
+
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    else:
```

Comment:
return ends the command. I recommend you remove the else and shift everything back by one indent. I try not to go into the right as much as possible to avoid spaghetti code